pkgs/development/tools/swc/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib
+, rustPlatform
+, fetchCrate
+}:
+
+rustPlatform.buildRustPackage {
+  name = "swc";
+  src = fetchCrate {
+    version = "0.91.11";
+    pname = "swc_cli";
+    sha256 = "sha256-vTmkuiMmvydLzMy+Qa0WST1mKSm2lFbG940VfKEeRI4=";
+  };
```

Comment:
```suggestion

  src = fetchCrate {
    pname = "swc_cli";
    version = "0.91.11";
    hash = "sha256-vTmkuiMmvydLzMy+Qa0WST1mKSm2lFbG940VfKEeRI4=";
  };
```

Reply:
and please inherit version

Reply:
`sha256` is a required argument for `fetchCrate`